src/Collection.js

```diff
@@ -52,6 +52,17 @@ module.exports = class Collection {
         return Promise.resolve(counter);
     }
 
+    updateOne(query, modifier, options) {
+        options = options || {};
+        options.multi = false;
+        return this.update(query, modifier, options);
```

Comment:
this could one liner `return this.update(query, modifier, Object.assign({}, options, { multi: false }));`